evennia/typeclasses/tests.py

```diff
@@ -22,6 +26,14 @@ def test_attrhandler(self):
         self.obj1.db.testattr = value
         self.assertEqual(self.obj1.db.testattr, value)
 
+        value = DictSubclass({"fo": "foo", "bar": "bar"})
+        self.obj1.db.testattr = value
+        self.assertEqual(self.obj1.db.testattr, value)
+
+        value = DictSubclass({"fo": "foo", "bar": "bar", "obj": self.obj2})
+        self.obj1.db.testattr = value
+        self.assertEqual(self.obj1.db.testattr, value)
+
```

Comment:
I'd like to also see tests of on-object inserts and reads here; so checking that you can do `self.obj1.db.testattr['fo'] = 'foo2'` and read data out from nested structures. 
Also would be useful to make sure that you can store a `DictSubclass` with another `DictSubclass` (and/or a regular dict by all means) nested inside it.

Reply:
ah great, thanks: I wasn't totally sure on how to cover all the cases

Reply:
done

Reply:
("resolving" so the red note disappear below :P)